src/api/getAllData.tsx

```diff
@@ -0,0 +1,272 @@
+            teamData.offenses += 1
+          }
+        }
+        if(today.getFullYear() - createdDate.getFullYear() == 0 || 
+          (today.getFullYear() - createdDate.getFullYear() == 1 && createdDate.getMonth() - today.getMonth() > 0)) {
+          if(vulns.severity == "CRITICAL"){
+
+            if (vulns.state == "OPEN"){
+              newRepo.critical += 1
+              teamData.vulnData.criticalNum += 1
+            }
+            
+            let i = createdIndex
+            do {
+              i %= 12
+              teamData.vulnData.critical[i] += 1
+              i += 1
+            } while(i != dismissedIndex && i != today.getMonth() + 1)
+
+            if(!seen.has(id)) {
+              if (vulns.state == "OPEN"){
+                orgData.vulnData.criticalNum += 1
+                topicsCriticalNum += 1
+              }
+
+              let i = createdIndex
+              do {
+                i %= 12
+                orgData.vulnData.critical[i] += 1
+                topicsCritical[i] += 1
+                i += 1
+              } while(i != dismissedIndex && i != today.getMonth() + 1) 
+            }
+          }
+          else if(vulns.severity == "HIGH"){
+
+            if (vulns.state == "OPEN"){
+              newRepo.high += 1
+              teamData.vulnData.highNum += 1
+            }
+
+            let i = createdIndex
+            do{
+              i %= 12
+              teamData.vulnData.high[i] += 1
+              i += 1
+            } while(i != dismissedIndex && i != today.getMonth() + 1)
+
+            if(!seen.has(id)) {
+              if (vulns.state == "OPEN"){
+                orgData.vulnData.highNum += 1
+                topicsHighNum += 1
+              }
+              let i = createdIndex
+              do {
+                i %= 12
+                orgData.vulnData.high[i] += 1
+                topicsHigh[i] += 1
+                i += 1
+              } while(i != dismissedIndex && i != today.getMonth() + 1) 
+            }
+          }
+          else if(vulns.severity == "MODERATE"){
+
+            if (vulns.state == "OPEN"){
+              newRepo.moderate += 1
+              teamData.vulnData.moderateNum += 1
+            }
+
+            let i = createdIndex
+            do {
+              i %= 12
+              teamData.vulnData.moderate[i] += 1
+              i += 1
+            } while(i != dismissedIndex && i != today.getMonth() + 1) 
+
+            if(!seen.has(id)) {
+              if (vulns.state == "OPEN"){
+                orgData.vulnData.moderateNum += 1
+                topicsModerateNum += 1
+              }
+
+              let i = createdIndex
+              do {
+                i %= 12
+                orgData.vulnData.moderate[i] += 1
+                topicsModerate[i] += 1
+                i += 1
+              } while(i != dismissedIndex && i != today.getMonth() + 1) 
+            }
+          }
+          if(vulns.severity == "LOW"){
```

Comment:
This should be an `else if` or make the `else if` above an `if`. It's better to stay consistent